api/overdrive.py

```diff
@@ -1008,13 +1064,31 @@ def update_licensepool_with_book_info(self, book, license_pool, is_new_pool):
 
     @classmethod
     def get_download_link(self, checkout_response, format_type, error_url):
+        """Extract a download link from the given response.
+
+        :param checkout_response: A JSON document describing a checkout-type
+           response from the Overdrive API.
+        :param format_type: The internal (Overdrive-facing) format type
+           that should be retrieved. 'x-manifest' format types are treated
+           as a variant of the 'x' format type -- Overdrive doesn't recognise
+           'x-manifest' and uses 'x' for delivery of both streaming content
+           and manifests.
+        :param error_url: Value to interpolate for the {errorpageurl}
+           URI template value.
+        """
         link = None
         format = None
         available_formats = []
+        if format_type in self.MANIFEST_INTERNAL_FORMATS:
+            use_format_type = format_type.replace("-manifest", "")
+            fetch_manifest = True
```

Comment:
This is where we convert from "overdrive-audiobook-manifest" (our internal name) to "overdrive-audiobook" (Overdrive's internal name).